<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Colletion</class>
 <widget class="QWidget" name="Colletion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>276</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>收藏夹</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(71, 71, 71);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_main">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_top">
     <item>
      <widget class="QPushButton" name="btn_back">
       <property name="text">
        <string>← 返回</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_title">
       <property name="styleSheet">
        <string>font-size:16px;font-weight:bold;</string>
       </property>
       <property name="text">
        <string>Rosebud的收藏夹</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(179, 179, 179);
color: rgb(20, 20, 20);</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_content">
      <attribute name="title">
       <string>内容</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_colletion">
      <attribute name="title">
       <string>收藏夹</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_tab">
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>234</width>
            <height>240</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="colletionLayout">
           <item>
            <widget class="QLabel" name="label_empty">
             <property name="styleSheet">
              <string>color:gray;font-size:14px;</string>
             </property>
             <property name="text">
              <string>暂无更多内容</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>200</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
